src/index.ts

```diff
@@ -3,18 +3,29 @@ import { toByteArray } from 'base64-js';
 if (typeof TextDecoder === 'undefined') {
   const nodeVer = typeof process !== 'undefined' && process.versions?.node;
   const nodeRequire = nodeVer
-    ? // @ts-ignore Isomorphism.
+    ? // @ts-expect-error Isomorphism.
       typeof __webpack_require__ === 'function'
-      ? // @ts-ignore Isomorphism.
+      ? // @ts-expect-error Isomorphism.
         __non_webpack_require__
-      : require
+      : // @ts-expect-error Isomorphism.
+        require
     : undefined;
-  // @ts-ignore Isomorphism.
-  global['TextDecoder'] = nodeRequire('util').TextDecoder;
+  globalThis['TextDecoder'] = nodeRequire('util').TextDecoder;
```

Comment:
`global` is legacy syntax; `globalThis` is [recommended](https://nodejs.org/api/globals.html#global)